lib/set.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-# :markup: markdown
```

Comment:
This usage fouled the definition list items (<tt># #foo:: Returns a foo.</tt>), so I removed it.  This required the other changes in the existing doc, up to line 57.